### PR TITLE
DOCSP-39857 changed configureQueryAnalyzer limitations

### PR DESCRIPTION
**Pull request by sonderdonk-mdb**
## DESCRIPTION
changed multi-tenant to shared tier and serverless, also added links 

## STAGING
https://preview-mongodbsonderdonkmdb.gatsbyjs.io/docs/DOCSP-39857/reference/command/configureQueryAnalyzer/

https://preview-mongodbsonderdonkmdb.gatsbyjs.io/docs/DOCSP-39857/reference/command/analyzeShardKey/ 

## JIRA
https://jira.mongodb.org/browse/DOCSP-39857

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=666c5c47ebce19f314e1cba0

## SELF-REVIEW CHECKLIST

- [ ] Does each file have 3-5 taxonomy facet tags?
  See the [taxonomy tagging instructions](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) and this [example PR](https://github.com/10gen/cloud-docs/pull/5042/files) 
- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## EXTERNAL REVIEW REQUIREMENTS

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/includes/cqa-limitations.rst [hunk 1]
```diff
@@ -1,8 +1,5 @@
-- You cannot run |CQA| on Atlas 
-  :atlas:`multi-tenant </build-multi-tenant-arch>`
-  configurations.
-- You cannot run |CQA| on 
-  standalone deployments.
+- You cannot run |CQA| on Atlas :ref:`shared clusters <shared-tier-cluster>` and :ref:`serverless instances <atlas-choose-serverless>`
```
Comment (kennethdyer) [inline source/includes/cqa-limitations.rst:1]: This line is a bit on the long side.  If you're on vim, highlight part of the line in visual mode then run `gw`.

On VS Code, the onboarding stuff calls for an extension called Rewrap.  Highlight the line and open the command pallet, then run Rewrap.  I'd set it to column 90.  That should automatically wrap to the lines you want.

Or, you could manually break it up.

**Review by kennethdyer — CHANGES_REQUESTED** [review]
Thanks for this @sonderdonk-mdb.  I left a comment for your consideration.

**Review by ratika-gandhi — APPROVED** [review]
LGTM

**Review by kennethdyer — APPROVED** [review]
Thanks @sonderdonk-mdb, LGTM.

**Pull request merged**